force-app/main/default/lwc/donationHistoryTable/__tests__/donationHistoryTable.test.js

```diff
@@ -0,0 +1,48 @@
+import { createElement } from 'lwc';
+import DonationHistoryTable from '../donationHistoryTable';
+const MOCK_DATA = require('./data/donationHistoryTableData.json');
+
+// Helper function to wait until the microtask queue is empty. This is needed for promise
+// timing when calling imperative Apex.
+function flushPromises() {
+    // eslint-disable-next-line no-undef
+    return new Promise((resolve) => setImmediate(resolve));
+  }
```

Comment:
This function already exists in the global scope for JEST tests here: https://github.com/SalesforceFoundation/NPSP/blob/18d9486df8929a219bf9de918e6938b4d347ef32/jest.setup.js#L5 , no need to redefine it here.